pre-docs/applying_manifest_blocks.md

```diff
@@ -9,7 +9,26 @@ Within a plan, you can use Bolt to apply blocks of Puppet code (manifest blocks)
 [Writing plans](writing_plans.md)
 
 **Note:** If you installed bolt as a ruby gem, you'll want to:
-1. Create a Puppetfile with the contents of [bolt's Puppetfile](../Puppetfile) at `~/.puppetlabs/bolt/Puppetfile`
+1. Create a Puppetfile with the contents of [bolt's Puppetfile](../Puppetfile) at `~/.puppetlabs/bolt/Puppetfile`:
```

Comment:
I don't think putting this inline is a good idea. I highly suspect it will result in us forgetting to update it.

side-note. The local modules should be left out of this.